blog: first draft of dashboard blog article

nicojs opened the pull request:
TODO:

- [x] Finish Stryker.NET implementation and update the [handbook](https://github.com/stryker-mutator/stryker-handbook/blob/master/dashboard.md)
- [x] Update the [Stryker.NET](https://dashboard.stryker-mutator.io/reports/github.com/stryker-mutator/stryker-net/master) and [Stryker4s](https://dashboard.stryker-mutator.io/reports/github.com/stryker-mutator/stryker4s/master) reports (dead links a.t.m)
- [x] Review blog article
- [x] Specifically mention Mull, InfectionPHP and Pitest.
- [x] Correct date on this blog article

hugo-vrijswijk commented:
@nicojs The Stryker4s report is up on the dashboard in master: [![Mutation testing badge](https://img.shields.io/endpoint?style=flat&url=https%3A%2F%2Fbadge-api.stryker-mutator.io%2Fgithub.com%2Fstryker-mutator%2Fstryker4s%2Fmaster)](https://dashboard.stryker-mutator.io/reports/github.com/stryker-mutator/stryker4s/master), you can edit the OP 😁

nicojs commented on src/blog/2019-12-17/blog.json:
```diff
@@ -0,0 +1,5 @@
+{
+    "title": "Show you're mutation testing report",
```
Like what? 🤷‍♂️

richardwerkman replied:
### Show your mutation testing report online using the stryker dashboard

or

### Stryker dashboard: host your mutation testing report in the cloud

richardwerkman commented on src/blog/2019-12-17/show-your-mutation-testing-report.pug:
```diff
@@ -0,0 +1,61 @@
+        
+        Getting up and running with the dashboard is simple. 
+
+        🎯 Navigate to https://dashboard.stryker-mutator.io and log in with your GitHub account.  
```
Shouldn't the link be clickable?

richardwerkman commented on src/blog/2019-12-17/show-your-mutation-testing-report.pug:
```diff
@@ -0,0 +1,61 @@
+        🎯 Navigate to https://dashboard.stryker-mutator.io and log in with your GitHub account.  
+        ✅ Enable the Stryker Dashboard for your repository.  
+        🔑 Configure Stryker to run with the "dashboard" reporter with your personal API key. See [the dashboard article on the Stryker handbook for the details](https://github.com/stryker-mutator/stryker-handbook/blob/master/dashboard.md)  
+        🏃‍♀️ Run Stryker.NET, Stryker4s or Stryker and upload your first report.
```
Maybe we should mention the dashboard reporter should be enabled when running?

richardwerkman commented on src/blog/2019-12-17/show-your-mutation-testing-report.pug:
```diff
@@ -0,0 +1,61 @@
+        We're always looking for feedback, please don't hesitate to create [an issue](https://github.com/stryker-mutator/stryker-dashboard/issues/new). 
+
+        The latest versions of Stryker, Stryker4s, and Stryker.NET all support the dashboard reporter.
+        We are also eating our own dog food here 🍴🐶🥫.
```
I like the saying, but it seems a bit random here. Maybe this could be the last header like:

## Eat your our own dog food 🍴🐶🥫.
The latest versions of Stryker, Stryker4s, and Stryker.NET all support the dashboard reporter.
We know our mutation score isn't perfect, so we welcome PR's to help improve our own Mutation Score. 
Improving tests is a good way to get to know a new code base.

nicojs commented:
Thanks Hugo and Richard for the feedback. I will update it .. maybe tomorrow.

@richardwerkman would you please update the handbook article?

nicojs commented:
@hugo-vrijswijk @richardwerkman I've updated the article based on your feedback.

I've also added a better introduction. Instead of letting a user fall off a cliff when they're not up-to-speed. 

Please take another look. I'll publish it after Christmas. ➡🎄
